src/components/ActionButtons.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <TodoButton v-if="!todo.done" @handler="() => $emit('handleDone', todo.id)"
```

Comment:
```suggestion
    <TodoButton v-if="!todo.done" @click="onDone"
```

Do this in other places.